### ISSUE TITLE
Subdomain takeover using mintlify.com

### ISSUE THREAD

**Issue by serundengsapi** [EdOverflow/can-i-take-over-xyz#413]
## Service name
mintlify.com

## Proof
![image](https://github.com/user-attachments/assets/d07755e8-db82-4173-aefe-f7fdfbf16a8a)

## Documentation

1. Create a account at mintlify.com
2. add a custom domain which pointing to `cname.vercel-dns.com`
3. When you already added a custom domain, you will receive the email like this it means the the subdomain successfully to takeover
![image](https://github.com/user-attachments/assets/d302af3c-701f-434b-a348-e8d02c0a8c16)

reference : https://mintlify.com/docs/settings/custom-domain

**Comment by marcelo321** [comment]
@serundengsapi the email I got asks for TXT record too, making this impossible to takeover. can u double-check?

**Comment by serundengsapi** [comment]
> @serundengsapi the email I got asks for TXT record too, making this impossible to takeover. can u double-check?

Yeah, when you adding a subdomain and got receive email ask for TXT record, it means the subdomain can't to takeover
